### DISCUSSION TITLE
How to get AddressableEntity?

### DISCUSSION THREAD

**Discussion by sczembor** [casper-network/condor-info#19] [Q&A] [answered]
How to get the `AddressableEntity` of a wallet, smart contract in a transaction data?

**Comment by sczembor** [comment]
`AddressableEntity` is the new way to refer to both accounts and contracts. Contracts are represented by `entity-contract-<hash>`, and accounts by either `entity-account-<hash>` or their public key in the transaction data. You can obtain the transaction data using this query:`casper-client get-transaction <transaction_hash> -n <node_address>`

**Reply by melpadden** [reply, marked as answer]
https://github.com/casper-network/condor-info/blob/main/articles/039-addressable_entity.md
